compiler/natives/doc.go

```diff
@@ -5,5 +5,9 @@
 // in src subfolder.
 package natives
 
+import (
+	_ "github.com/shurcooL/vfsgen" // Force go.mod to require this package
+)
+
```

Comment:
I tried it with the build tag and an unused directory, and go.mod didn't seem to pick it up.  This seemed like a good spot.

Reply:
SGTM.